### ISSUE TITLE
ID LibreView

### ISSUE THREAD

**Issue by Geniy83** [j-kaltes/Juggluco#121]
Hello. Is it possible to find out your account ID in LibreView from a browser? To then manually enter it into Juggluco

**Comment by j-kaltes** [comment]
Yes, that is possible. You can use the java program from https://www.juggluco.nl/Juggluco/libre3/nolibreview.html with the UUID that comes after "_https://www.libreview.com/reports/_" to generate the Account ID number.
